README.md

```diff
@@ -13,25 +13,26 @@ Currently supports:
 - [Webpack](https://webpack.js.org/)
 - [esbuild](https://esbuild.github.io/)
 - [Rspack](https://www.rspack.dev/) (⚠️ experimental)
+- [Farm](https://farm-fe.github.io/)
```

Comment:
Should we mark it as experimental?

Reply:
Farm has not yet released 1.0. it may follow the use of users to modify the plugin system. Thank you for your reminder.